examples/step-78/step-78.cc

```diff
@@ -747,12 +744,14 @@ namespace BlackScholesSolver
         // vector:
         vmult_result.reinit(dof_handler.n_dofs());
         forcing_terms.reinit(dof_handler.n_dofs());
-        while (time < maturity_time)
+        for (unsigned int timestep_number = 0; timestep_number < n_time_steps;
+             ++timestep_number)
           {
             time += time_step;
-            ++timestep_number;
-            std::cout << "Time step " << timestep_number << " at t=" << time
-                      << std::endl;
+
+            if (timestep_number % 1000 == 0)
```

Comment:
```suggestion
            if (timestep_number % 1000 == 0 || timestep_number == n_time_steps - 1)
```
I think it would be nice to print out the very last time step too. What do you think? I don't feel strongly about this.

Reply:
I'd rather not because then we have

1000
2000
3000
4000
4999

which is aesthetically unappealing.

Reply:
My argument is that you will have the first and last time step in `cout`, so that you can be sure that your cycle succeeded. But as I said I don't feel strong about it :-)